packages/console/src/pages/SignInExperience/components/Preview.tsx

```diff
@@ -106,15 +106,15 @@ const Preview = ({ signInExperience, className }: Props) => {
       signInExperience: {
         ...signInExperience,
         socialConnectors,
+        forgotPassword: {
+          email: hasEmailConnector,
+          sms: hasSmsConnector,
+        },
       },
       language,
       mode,
       platform: platform === 'desktopWeb' ? 'web' : 'mobile',
       isNative: platform === 'mobile',
```

Comment:
Maybe it would be better to use a return type to guard the return value?

Reply:
It is the response body type of the sign-in-experience API. Unfortunately, we do not have a type for the API response type yet. It might be a bigger topic